### ISSUE TITLE
 Upgrade script error: pathspec 'undefined' did not match any file(s) known to git

### ISSUE THREAD

**Issue by markusd1984** [adaptlearning/adapt_authoring#2474]
## Expected Behaviour
Upgrading Adapt authoring tool v0.10.1 to v0.10.2 without errors.                                                                                        
## Actual Behaviour
```
/home/nonroot/adapt_authoring switched to revision v0.10.2
Node dependencies installed successfully.
Web application built successfully.
Server has been updated successfully!
Adapt authoring tool upgraded to v0.10.2
Failed to upgrade /temp/5dc2c6f50cf3b242e1988676/adapt_framework to undefined
ERROR: error: pathspec 'undefined' did not match any file(s) known to git.
```
## Steps to Reproduce
node upgrade.js

## Versions
  - Authoring Tool Version: v0.10.1
  - Framework Version: 4.4.1.
  - Node.js Version: v8.10.0
  - MongoDB Version: v3.6.3
  - Operating System: Ubuntu 18.04 LTS
  - Browser: n/A

**Comment by tomgreenfield** [comment]
Note: you have to be already on the latest framework version to replicate.

**Comment by tomgreenfield** [comment]
This should've been fixed by #2549.
